src/lib/components/ResponseMultipleChoice/index.js

```diff
@@ -0,0 +1,185 @@
+import {html} from "lit-element";
+import {BaseResponseElement} from "../BaseResponseElement";
+import "./_styles.scss";
+
+/* eslint-disable require-jsdoc */
+class ResponseMultipleChoice extends BaseResponseElement {
+  static get properties() {
+    return {
+      id: {type: String, reflect: true},
+      cardinality: {type: String},
```

Comment:
If you set `type: Number` then you can just treat this as a number (below you check for _string_ "1").

Reply:
Actually I remember that this is allows a range. Perhaps add a comment?

Reply:
Yup! Good call.